internal/controller/account_resolver.go

```diff
@@ -39,6 +40,15 @@ type authGwErrorResponse struct {
 	} `json:"errors"`
 }
 
+func (this authGwErrorResponse) String() string {
```

Comment:
This implements the `Stringer` interface on the authGwErrorResponse object/struct.  The `Stringer` interface basically consists of a `String()` method.  This us to print out an instance of the authGwErrorResponse object and the instance will convert itself into a string.